client/src/App.js

```diff
@@ -1,13 +1,31 @@
-import './App.css';
-import ShowAllProjects from './components/ShowAllProjects';
-import allProjects from './db';
+import './App.css'
+// import ShowAllProjects from './components/showAllProjects/ShowAllProjects'
+import allProjects from './db'
+import { useState, useEffect } from 'react'
+import AddProject from './pages/AddProject'
+import AllProjects from './pages/AllProjrcts'
+import { Routes, Route } from 'react-router-dom'
+
 
 
 function App() {
+    const [newPr, setNewPr] = useState(allProjects)
```

Comment:
For better understanding of your code it would be better to not work with abbreviations. Instead of `newPr` and `setNewPr` you should work with `new Project` and `setNewProject`.